src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import { createUserClient } from "~/src/util/get-github-client-config";
+
+export const GithubCreateBranchPost = async (req: Request, res: Response): Promise<void> => {
+	const { githubToken, jiraHost } = res.locals;
+	const { owner, repo, sourceBranchName, newBranchName } = req.body;
+
+	if (!githubToken || !jiraHost) {
+		res.sendStatus(401);
+		return;
+	}
+
+	if (!owner || !repo || !sourceBranchName || !newBranchName) {
+		res.status(400).json({ err: "Missing required data." });
+		return;
+	}
+
+	try {
+		const gitHubUserClient = await createUserClient(githubToken, jiraHost, req.log, undefined);
```

Comment:
Passing `undefined` would only work for GitHub cloud, what about GitHub servers? 
For GHES, we will be passing a `gitHubServerAppId`(primary key), which can be passed here.

Reply:
yeah, have left out the GHES stuff intentionally until we plug it in proper.

Reply:
/nit Please add a TODO so that its not lost or forgotten...

Reply:
word